runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -137,12 +135,25 @@ private ClientUI(RuneliteProperties properties, Applet client)
 		this.trayIcon = setupTrayIcon();
 
 		init();
-		pack();
-		new TitleBarPane(this.getRootPane(), (SubstanceRootPaneUI)this.getRootPane().getUI()).editTitleBar(this);
 		setTitle(null);
 		setIconImage(ICON);
 		setLocationRelativeTo(getOwner());
```

Comment:
This probably needs to be moved after setVisible(true), because now, after the window is displayed, it is not centered properly (not sure actually if that will help, but it should, and the not centering issue is a bit annoying).